v63003/inref/gtm8680.m

```diff
@@ -19,24 +19,24 @@
 	set maxit=0
 	set maxit2=0
 	set minit=0
-	set minit2=0
-	set ^its($increment(^i))=0
+	set its($increment(it))=0
 	set start=$h,cur=start,prev=start,previ=0,t=$$^difftime(cur,start)
-	for i=1:1 do  quit:(t>15)
+	for i=1:1 do  quit:(t>30)
 	. lock ^a(i#max):0
 	. set cur=$h,diff=$$^difftime(cur,prev)  do
 	. if diff>0 do  set prev=cur,previ=i,t=$$^difftime(cur,start)
-	. . set ^its($increment(^i))=i-previ
-	. . if t=2 set maxit=i-previ,minit=i-previ,maxit2=i-previ,minit2=i-previ
-	. . if t>=2 do
-	. . . if i-previ>maxit2 set maxit2=i-previ
-	. . . if i-previ<minit2 set minit2=i-previ
-	. . . if i-previ>maxit set maxit2=maxit,maxit=i-previ
-	. . . if i-previ<minit set minit2=minit,minit=i-previ
+	. . set its($increment(it))=i-previ
+	. . if t=1 set maxit=i-previ,minit=i-previ,maxit2=i-previ
```

Comment:
Can this line be moved outside the for loop to initialize maxit and minit to 0.

Reply:
I meant initializing minit to a HUGE number. Say (2**32)

Reply:
Or do not initialize minit outside the for loop but initialize it whenever maxit gets initialized inside the for loop. That is fine since this is a min-after-max algorithm.